server/src/main/java/org/elasticsearch/bootstrap/BootstrapUtil.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.bootstrap;
+
+import org.elasticsearch.cli.Terminal;
+import org.elasticsearch.common.settings.KeyStoreWrapper;
+import org.elasticsearch.common.settings.SecureSettings;
+import org.elasticsearch.common.settings.SecureString;
+import org.elasticsearch.env.Environment;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Utilities for use during bootstrap. This is public so that tests may use these methods.
+ */
+public class BootstrapUtil {
+
+    /**
```

Comment:
Trivially, you could optionally add a private no-args constructor to BootstrapUtil, since it would appear to be a non-instantiable class exposing static utility methods.